SConstruct

```diff
@@ -48,7 +48,7 @@ import distutils.dir_util
 
 EnsureSConsVersion( 3, 0, 2 ) # Substfile is a default builder as of 3.0.2
 
-if locale.getpreferredencoding() != "UTF-8" :
+if locale.getpreferredencoding().upper() != "UTF-8" :
```

Comment:
I don't know if it matters or not, but I think `codecs.lookup( locale.getpreferredencoding() ).name` is the official way to get a canonical encoding name, so might be better than using `upper()`?